### ISSUE TITLE
Misc minor changes (updated regularlyish)

### ISSUE THREAD

**Issue by JasonP01** [xpdustry/imperium#18]
I will open/close this when things are added/changed or whatever

Discord stuff:

Currently the user ban/kick notif we get in discord is rather ~~shit~~ it should 1000% tell us for the votekicks who voted, and have the user id in a separate category thing (so its easy to copy on mobile). also need to add time parser or whatever PT1HR is terrible

Bot command descriptions for things, its very confusing what is going on, things are labeled in a way you understand but we dont. for instance what is the whole `punishment` commands, seems very weird and i cant seem to figure it out. things should be labeled so the discord interface isnt terrible to use as its used more than ingame stuff are during moderation

Ingame stuff:

None currently

**Comment by JasonP01** [comment]
yes i am not saying this in a more professional manner, if you would like me to make these issues in a more professional manner please say so, and also please dont close this yourself, i need to ability to reopen it so i dont make millions of issues for minor things (if you would like it closed ping me on discord, it will be handled within 8 hours unless im asleep)

**Comment by phinner** [comment]
> Currently the user ban/kick notif we get in discord is rather shit it should 1000% tell us for the votekicks who voted

I am already working on it

> Bot command descriptions for things, its very confusing what is going on, things are labeled in a way you understand but we dont. for instance what is the whole punishment commands, seems very weird and i cant seem to figure it out. things should be labeled so the discord interface isnt terrible to use as its used more than ingame stuff are during moderation

:skull: I forgor about translating them.

> Currently some players like to think they are funny and use other server nonsense or have stupid things in their name, such as fish <+> (their trusted rank) or [ADMIN] which is obviously fake, maybe add a thing to add to their name like "Look at this dude" or something kinda funny, fish adds "SUSSY IMPOSTER" to users who fake stuff so maybe something like that.

I'll need to rewrite the player prefix system for that, which means, not now.

> yes i am not saying this in a more professional manner, if you would like me to make these issues in a more professional manner please say so

Yes, I do

> i need to ability to reopen it so i dont make millions of issues for minor things 

That's the point of issues, focusing on one problem to solve. Having multiple in one is annoying.

**Comment by JasonP01** [comment]
> Yes, I do

Noted in future issues i will be more professional, cant guarentee comments will be professional, id rather keep my comments on things personal

> That's the point of issues, focusing on one problem to solve. Having multiple in one is annoying.

Also okay, my mindset having a billion seperate things are more annoying than 1 thing with a billion all together, ill keep this in mind.

As for the other issue labled with #suggestions from discord, would you like me to put any ones in particular in seperate issues, making 11± issues is going to be tedious, if not i will just close it

**Comment by JasonP01** [comment]
For the suggestion im talking about #6

**Comment by phinner** [comment]
Keep it

**Comment by phinner** [comment]
Since it has been reduced to one issue, closing.
